chat-info.cpp

```diff
@@ -39,7 +39,7 @@ std::string getChatName(const td::td_api::chat &chat)
 GHashTable *getChatComponents(const td::td_api::chat &chat)
 {
     char name[32];
-    snprintf(name, sizeof(name)-1, "chat%lld", (long long)chat.id_);
```

Comment:
There should have been a warning here, because chat.id_ is signed.

Don't fix it though, I'm fixing it myself, since it's just a simple mass-replace.